PokeAlarm/FacebookPage/FacebookPageAlarm.py

```diff
@@ -23,15 +23,20 @@ class FacebookPageAlarm(Alarm):
 
     _defaults = {
         'pokemon': {
-            'message': "A wild <pkmn> has appeared! Available until <24h_time> (<time_left>).",
-            'link': "<gmaps>"
+            'message': "A wild <pkmn> has appeared!.",
+            'picture': "https://raw.githubusercontent.com/kvangent/PokeAlarm/master/icons/<pkmn_id>.png",            
+            'link': "<gmaps>",
+            'name': "<pkmn>", 
+            'description': "Available until <24h_time> (<time_left>)"
         },
         'pokestop': {
             'message': "Someone has placed a lure on a Pokestop! Lure will expire at <24h_time> (<time_left>).",
```

Comment:
Can we get some defaults for the other alert types added? Suggestions:

pokestop:
- Message: `"Someone has placed a lure on a Pokestop!"`
- Name: `"Lured Pokestop"`
- Description: `"Lure will expire at <24h_time> (<time_left>)."`
- Caption: `None` <- Just so it is clear this is an option

Gym:
- Message: `"Team <new_team> has claimed a new gym! "`
- Name: `"It is now controlled by <new_team>."`
- Description: `"Lure will expire at <24h_time> (<time_left>)."`
- Caption: `None` <- Just so it is clear this is an option